core/rust/qdbr/src/parquet_read/meta.rs

```diff
@@ -199,7 +229,7 @@ mod tests {
 
         for (i, col) in meta.columns.iter().enumerate() {
             let (col_type, _, name) = cols[i];
-            assert_eq!(col.typ, to_storage_type(col_type));
+            assert_eq!(col.typ, col_type);
```

Comment:
do we have something for ColumnType::Char ?

Reply:
I don't understand the question, can you rephrase it?